cockroachcloud/serverless-faqs.md

```diff
@@ -0,0 +1,95 @@
+toc: true
+---
+
+This page answers the frequently asked questions about CockroachCloud Serverless (beta) and CockroachCloud Dedicated.
+
+<div class="filters clearfix">
+    <a href="serverless-faqs.html"><button class="filter-button page-level current">CockroachCloud Serverless</button></a>
+    <a href="frequently-asked-questions.html"><button class="filter-button page-level">CockroachCloud Dedicated</button></a>
+</div>
+
+## General
+
+### What is CockroachCloud Serverless (beta)?
+
+CockroachCloud Serverless (beta) delivers free CockroachDB clusters for you and your organization. It is a managed instance of CockroachDB that removes the friction of initial cluster sizing and auto-scales based on your application traffic.
+
+### How do I start using CockroachCloud Serverless (beta)?
+
+To get started with CockroachCloud Serverless (beta), <a href="https://cockroachlabs.cloud/signup?referralId=docs_serverless_faq" rel="noopener" target="_blank">sign up for a CockroachCloud account</a>, click **Create Cluster**, then click **Create your free cluster**. Your cluster will be ready in 20-30 seconds. For more information, see [**Quickstart**](quickstart.html).
+
+### What are the usage limits of Cockroach Cloud Serverless (beta)?
+
+Free clusters have a limit of 100M Request Units per month and 5GB of storage, but the performance is throttled to a maximum of 100 RUs/second. Paid clusters have access to the same resources with no limitations in addition to the amount you pay for.
+
+### What is a Request Unit?
+
+All resource usage in CockroachCloud Serverless (beta) is measured in Request Units, or RUs. RUs represent the compute and I/O resources used by a read or a write query. All database operations cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs.
+
+### Do I have to pay for CockroachCloud Serverless (beta)?
+
+No, you can create a Serverless cluster that is free forever. If you choose to create a paid cluster, you will only be charged for the resources you use up to your spend limit.
+
+### What can I use CockroachCloud Serverless (beta) for?
+
+Free CockroachCloud Serverless (beta) clusters can be used for proofs-of-concept, toy programs, or to use while completing [Cockroach University](https://www.cockroachlabs.com/cockroach-university/).
+
+For examples of applications that use free clusters, check out the following [Hack the North](https://hackthenorth.com/) projects:
+
+- [flock](https://devpost.com/software/flock-figure-out-what-film-to-watch-with-friends)
+- [mntr.tech](https://devpost.com/software/mntr-tech)
+- [curbshop.online](https://devpost.com/software/curbshop-online)
+
+Paid Serverless clusters have higher performance, additional features, and the ability to scale according to your needs. They can be used for all kinds of production applications.
```

Comment:
What's the definition of a paid cluster? If we're talking about import/export then that's actually a function of adding a credit card to your org, not a function of setting the cluster's budget.

Reply:
Paid cluster is non-zero budget. Fixed this and added more callouts about adding a credit card.